buildContainers.nf

```diff
@@ -211,8 +212,8 @@ def helpMessage() {
   log.info "       Default: all"
   log.info "       Possible values:"
   log.info "         all, fastqc, freebayes, gatk, igvtools, multiqc, mutect1"
-  log.info "         picard, qualimap, r-base, runallelecount, sarek"
-  log.info "         snpeff, snpeffgrch37, snpeffgrch38, vepgrch37, vepgrch38"
+  log.info "         picard, qualimap, r-base, runallelecount, sarek, snpeff"
```

Comment:
We have fastqc, multiqc, qualimap, vcftools, as separate containers: these are all QC tools, is it possible to keep them separate, or can we merge these containers sometime?

Reply:
That could be a good idea, I'll think about it

Reply:
#569